test/obb/events/game_events_test.clj

```diff
@@ -21,29 +21,20 @@
   (let [system (-> (game-events-system) component/start)
```

Comment:
Does this really become more readable with the threading macro? I think it would be more to have `(component/start (game-events-system))`. Feel free to ignore if you don't agree.